RELEASE_NOTES.rst

```diff
@@ -92,6 +92,7 @@ installed. Please install Hive Provider > 5.1.0 when using those macros.
 
 New Features
 ^^^^^^^^^^^^
+- AIP-50 Trigger DAG UI Extension with Flexible User Form Concept (`AIP-50 <https://github.com/apache/airflow/pulls?q=is%3Apr+is%3Amerged+label%3AAIP-50+milestone%3A%22Airflow+2.6.0%22>`_)
```

Comment:
We can just show https://github.com/apache/airflow/pull/27063
the bug fix of https://github.com/apache/airflow/pull/29376 can be skipped from change log because it's not a fix that relevant for the users. it's something found and addressed before release.

Reply:
For me, since the link above shows the 2 PRs, it's better than showing one and skipping the other. Both will have credits for contributing to the change

Reply:
Hmm. I think having two prs linke is the only way to go. Something like  `(#27063, #29376)`. 

The problem with labels is that they are not persistent. I can imagine we do cleanup in GitHub and remove old unused labels a year from now. Should it make more difficult to see which PRs were part of the change when users look for it in 2 years ? 

We are deliberately using PR# instead of ISSUE# (and have the rule that we do not have to have the issue) precisely for the reason to keep it persistently stored in our git history. Every single PR# we merge and display here is not only something that will link you directly to the PR on GitHub, but it is also always (our squash & merge workflow guarantees that) part of the commit message of the merged PR. 

This is important for the longevity of the Foundation PMCs. As an ASF PMC we have to be prepared to leave GitHub if needs be. If we decide tomorrow that we move to GitLab for example, we should be able to move our git repo there and the context shoudl move with us (it's is mirrored in GitBox service that is run by the ASF infrastructure). So if the release notes will say ` `(#27063,#29376)`, then you will be able to find the very commits it refers by searching commit messaes in GitLab. In such case we will have no idea whatsoever what `is%3Apr+is%3Amerged+label%3AAIP-50+milestone%3A%22Airflow+2.6.0%22` is all about.

This is also why I try to promote and encourage putting a lot of context in the commit messages and why we move decision making from GitHub issues to devlist for important matters.